src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
```

Comment:
You are using the '*' selector which impacts performance. It's better to set styles only for elements that require them. (CHECKLIST: Don't use '*' selector (it impacts performance))